tests/webapp/api/test_jobs_api.py

```diff
@@ -374,3 +375,16 @@ def test_list_similar_jobs(webapp, eleven_jobs_stored, jm):
     assert isinstance(similar_jobs['results'], list)
 
     assert len(similar_jobs['results']) == 3
+
+
+def test_job_create(webapp, test_repository, test_user, eleven_job_blobs, monkeypatch, jm):
+    monkeypatch.setattr(JobsViewSet, 'permission_classes', ())
```

Comment:
I wonder if we should use the `mock_post_json` fixture instead, and post using the Python client, like for the artifact api tests? That way we're testing client & API:
https://github.com/mozilla/treeherder/blob/8c1e8f9fccea18b1606c7bb3ea9e2808a808a7af/tests/webapp/api/test_artifact_api.py#L68-L88